### PR TITLE
Write locations in "Driver not found in any of the locations"

### PR DESCRIPTION
**Pull request by SimonCropp**
To help debug when something goes wrong.

### CODE REVIEW

**Comment by SimonCropp** [comment]
note that we had this today. the reason was we were pulling in `<PackageReference Include="Verify.Playwright" Version="0.11.0" />` but not an explicit  `<PackageReference Include="PlaywrightSharp" Version="0.192.0" />`. so the `.playwright` directory wasnt created. adding the explicit PlaywrightSharp ref fixed it. this wasnt required in v 0.180. not sure if it a bug or a qwerk of nuget. but just a heads up

**Comment by SimonCropp** [comment]
guessing it has something to do with no `PlaywrightSharp.props` in 192

**Comment by kblok** [comment]
I'm good with this change. Regarding your comment about the `.playwright`.  That's a known limitation. The output project needs to reference Playwright in order to get the `.playwright`folder.
I'm open to seeing a PR improving that.

**Review by kblok — APPROVED** [review]

File: src/PlaywrightSharp/Transport/Connection.cs [hunk 1]
```diff
@@ -258,20 +258,23 @@ private static string GetExecutablePath()
...
+                driversPath,
+                RuntimeInformation.IsOSPlatform(OSPlatform.OSX) || RuntimeInformation.IsOSPlatform(OSPlatform.Linux) ? "playwright.sh" : "playwright.cmd");
+
+            if (File.Exists(fallbackBinPath))
+            {
+                return fallbackBinPath;
             }
 
-            return executableFile;
+            throw new PlaywrightSharpException($@"Driver not found in any of the locations. Tried:
```
Comment (avodovnik) [inline src/PlaywrightSharp/Transport/Connection.cs:275]: I'm personally not a huge fan of having exceptions be in multiple lines.

**Review by avodovnik — APPROVED** [review]

**Pull request merged**